### ISSUE TITLE
shinyFiles - 'Warning in file.info(files)' on windows 10 

### ISSUE THREAD

**Issue by OmaymaS** [ropenscilabs/pkginspector#17]
I get warnings when I run the shiny app on windows 10. I guess it is coming from `shinyFiles::getVolumes()`

```
Warning in file.info(files) :
  cannot open file 'C:/hiberfil.sys': The process cannot access the file because it is being used by another process
Warning in file.info(files) :
  cannot open file 'C:/pagefile.sys': The process cannot access the file because it is being used by another process
Warning in file.info(files) :
  cannot open file 'C:/PerfLogs': Access is denied
Warning in file.info(files) :
  cannot open file 'C:/swapfile.sys': The process cannot access the file because it is being used by another process
Warning in file.info(files) :
  cannot open file 'C:/System Volume Information': Access is denied
Warning in file.info(files) :
  cannot open file 'C:/$Recycle.Bin/S-1-5-18': Access is denied
```

I guess the same issue was raised in issue [57](https://github.com/thomasp85/shinyFiles/issues/57) in `shinyFiles` package.

Anyone else faced this with Windows or any other OS?

@jtr13

**Comment by boshek** [comment]
I can confirm that I am getting the same issue. 

<details> <summary><strong>Session Info</strong></summary>

```r
Session info --------------------------------------------------------------------------------------------------------
 setting  value                       
 version  R version 3.5.0 (2018-04-23)
 system   x86_64, mingw32             
 ui       RStudio (1.1.383)           
 language (EN)                        
 collate  English_United States.1252  
 tz       America/Los_Angeles         
 date     2018-07-04                  

Packages ------------------------------------------------------------------------------------------------------------
 package      * version    date       source                                  
 assertthat     0.2.0      2017-04-11 CRAN (R 3.5.0)                          
 backports      1.1.2      2017-12-13 CRAN (R 3.5.0)                          
 base         * 3.5.0      2018-04-23 local                                   
 base64enc      0.1-3      2015-07-28 CRAN (R 3.5.0)                          
 bindr          0.1.1      2018-03-13 CRAN (R 3.5.0)                          
 bindrcpp       0.2.2      2018-03-29 CRAN (R 3.5.0)                          
 callr          2.0.4      2018-05-15 CRAN (R 3.5.1)                          
 clisymbols     1.2.0      2017-05-21 CRAN (R 3.5.0)                          
 codetools      0.2-15     2016-10-05 CRAN (R 3.5.0)                          
 commonmark     1.5        2018-04-28 CRAN (R 3.5.0)                          
 compiler       3.5.0      2018-04-23 local                                   
 covr           3.1.0      2018-05-18 CRAN (R 3.5.0)                          
 crayon         1.3.4      2017-09-16 CRAN (R 3.5.0)                          
 cyclocomp      1.1.0      2016-09-10 CRAN (R 3.5.0)                          
 datasets     * 3.5.0      2018-04-23 local                                   
 debugme        1.1.0      2017-10-22 CRAN (R 3.5.0)                          
 desc           1.2.0      2018-05-01 CRAN (R 3.5.1)                          
 devtools     * 1.13.5     2018-02-18 CRAN (R 3.5.0)                          
 digest         0.6.15     2018-01-28 CRAN (R 3.5.0)                          
 dplyr          0.7.6      2018-06-29 CRAN (R 3.5.1)                          
 evaluate       0.10.1     2017-06-24 CRAN (R 3.5.0)                          
 functionMap    1.0.0      2018-05-23 Github (MangoTheCat/functionMap@65a8ecc)
 gh             1.0.1      2017-07-16 CRAN (R 3.5.0)                          
 git2r          0.21.0     2018-01-04 CRAN (R 3.5.0)                          
 glue           1.2.0      2017-10-29 CRAN (R 3.5.0)                          
 goodpractice   1.0.2      2018-05-02 CRAN (R 3.5.0)                          
 graphics     * 3.5.0      2018-04-23 local                                   
 grDevices    * 3.5.0      2018-04-23 local                                   
 here           0.1        2017-05-28 CRAN (R 3.5.0)                          
 htmltools      0.3.6      2017-04-28 CRAN (R 3.5.0)                          
 htmlwidgets    1.2        2018-04-19 CRAN (R 3.5.0)                          
 httpuv         1.4.4.1    2018-06-18 CRAN (R 3.5.1)                          
 httr           1.3.1      2017-08-20 CRAN (R 3.5.0)                          
 igraph         1.2.1      2018-03-10 CRAN (R 3.5.0)                          
 jsonlite       1.5        2017-06-01 CRAN (R 3.5.0)                          
 knitr          1.20       2018-02-20 CRAN (R 3.5.0)                          
 later          0.7.3      2018-06-08 CRAN (R 3.5.1)                          
 lazyeval       0.2.1      2017-10-29 CRAN (R 3.5.0)                          
 lintr          1.0.2      2017-11-08 CRAN (R 3.5.0)                          
 magrittr       1.5        2014-11-22 CRAN (R 3.5.0)                          
 memoise        1.1.0      2017-04-21 CRAN (R 3.5.0)                          
 methods      * 3.5.0      2018-04-23 local                                   
 mime           0.5        2016-07-07 CRAN (R 3.5.0)                          
 pillar         1.2.3      2018-05-25 CRAN (R 3.5.1)                          
 pkgconfig      2.0.1      2017-03-21 CRAN (R 3.5.0)                          
 pkginspector * 0.0.1      <NA>       local                                   
 praise         1.0.0      2015-08-11 CRAN (R 3.5.0)                          
 processx       3.1.0      2018-05-15 CRAN (R 3.5.1)                          
 promises       1.0.1      2018-04-13 CRAN (R 3.5.1)                          
 pryr           0.1.4      2018-02-18 CRAN (R 3.5.0)                          
 purrr          0.2.5      2018-05-29 CRAN (R 3.5.0)                          
 R6             2.2.2      2017-06-17 CRAN (R 3.5.0)                          
 rcmdcheck      1.2.1      2016-09-28 CRAN (R 3.5.0)                          
 Rcpp           0.12.17    2018-05-18 CRAN (R 3.5.1)                          
 rematch        1.0.1      2016-04-21 CRAN (R 3.5.0)                          
 remotes        1.1.1.9000 2018-05-23 Github (MangoTheCat/remotes@e78d84e)    
 reprex         0.2.0      2018-06-22 CRAN (R 3.5.1)                          
 rex            1.1.2      2017-10-19 CRAN (R 3.5.0)                          
 RJSONIO        1.3-0      2014-07-28 CRAN (R 3.5.0)                          
 rlang          0.2.1      2018-05-30 CRAN (R 3.5.0)                          
 rmarkdown      1.10       2018-06-11 CRAN (R 3.5.1)                          
 roxygen2       6.0.1      2017-02-06 CRAN (R 3.5.0)                          
 rprojroot      1.3-2      2018-01-03 CRAN (R 3.5.0)                          
 rstudioapi     0.7        2017-09-07 CRAN (R 3.5.0)                          
 sankey         1.0.2      2017-10-22 CRAN (R 3.5.0)                          
 shiny        * 1.1.0      2018-05-17 CRAN (R 3.5.1)                          
 shinyFiles   * 0.6.2      2016-08-28 CRAN (R 3.5.1)                          
 simplegraph    1.0.0      2015-12-10 CRAN (R 3.5.0)                          
 stats        * 3.5.0      2018-04-23 local                                   
 stringi        1.1.7      2018-03-12 CRAN (R 3.5.0)                          
 stringr        1.3.1      2018-05-10 CRAN (R 3.5.0)                          
 swtext         0.0.1      2017-06-26 Github (boshek/swtext@4261578)          
 testthat     * 2.0.0      2017-12-13 CRAN (R 3.5.0)                          
 tibble         1.4.2      2018-01-22 CRAN (R 3.5.0)                          
 tidyr          0.8.1      2018-05-18 CRAN (R 3.5.0)                          
 tidyselect     0.2.4      2018-02-26 CRAN (R 3.5.0)                          
 tools          3.5.0      2018-04-23 local                                   
 usethis      * 1.3.0      2018-02-24 CRAN (R 3.5.0)                          
 utils        * 3.5.0      2018-04-23 local                                   
 visNetwork   * 2.0.4      2018-06-14 CRAN (R 3.5.1)                          
 whoami         1.1.2      2017-10-20 CRAN (R 3.5.0)                          
 withr          2.1.2      2018-03-15 CRAN (R 3.5.0)                          
 xml2           1.2.0      2018-01-24 CRAN (R 3.5.0)                          
 xmlparsedata   1.0.1      2016-06-18 CRAN (R 3.5.0)                          
 xtable         1.8-2      2016-02-05 CRAN (R 3.5.1)                          
 yaml           2.1.19     2018-05-01 CRAN (R 3.5.1)
```
</details>
